.github/workflows/scripts/create-cluster.sh

```diff
@@ -49,27 +50,32 @@ case $(cluster_status "$CNAME") in
 "")
     gh_summary "Cluster $CNAME doesn't exist."
     ;;
-1)
+0|FAILED)
+    # Cluster exists already.
+    infra_status_summary "$CNAME" "Cluster failed to create"
+    exit 0
```

Comment:
In the current state, infractl get for a cluster that failed to create will have no .Status field which will result in a  "null" value from `jq` and thus fall through to the `*)` case and it will still try to create the cluster. I'd say we want to keep the existing behavior i.e. do not exit 0 here. A failed cluster might be due to something that was rectified and we are trying again.